codegen/model/resource.go

```diff
@@ -56,6 +56,9 @@ type Group struct {
 	// Should we generate kubernetes manifests?
 	RenderManifests bool
 
+	// Should we generate validating schemas for CRDs?
```

Comment:
Do you think this should be true or false by default? I can't decide

Reply:
Given that there may still be protobuf constraints with the cue parser, I think it should be false by default so that it doesn't break code generation for downstream projects that upgrade skv2.

Reply:
Makes sense to me